package.json

```diff
@@ -11,7 +11,8 @@
   "files": [
     "dist"
   ],
-  "main": "./dist/ovh-angular-responsive-popover.js",
+  "main": "./dist/cjs/ovh-angular-responsive-popover.js",
```

Comment:
Could you please run: `npx sort-package-json`? :)

Reply:
done